Tests/SonarQube.MSBuild.Tasks.UnitTests/WriteProjectInfoFileTests.cs

```diff
@@ -211,7 +213,7 @@ public void WriteProjectInfoFile_MissingProjectGuid()
         }
 
         [TestMethod]
-        [Description("Tests that project info files are created for unrecognised languages")]
+        [Description("Tests that project info files are created for unrecognized languages")]
```

Comment:
These changes clutter the PR change log. (It's not a big deal, but next time let's put these into separate PRs.)